src/Foundation/NSItemProvider.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Threading.Tasks;
+using XamCore.CloudKit;
+
+namespace XamCore.Foundation 
+{
+#if MONOMAC && XAMCORE_2_0 // Only 64-bit on mac
+	public partial class NSItemProvider
+	{
+#if !XAMCORE_4_0
```

Comment:
Since we don't plan on keeping this API let's discourage it's usage with an `[Obsolete ("Please use ... API")]`